kitsune/users/admin.py

```diff
@@ -33,12 +33,8 @@ class ProfileAdmin(admin.ModelAdmin):
     form = ProfileAdminForm
     list_display = ['full_user']
     list_select_related = True
-    readonly_fields = ['user']
```

Comment:
Having this be read-only seems good. No?

Reply:
It does but by enabling the add permissions, you cannot select a user for the new profile as the field is readonly.

Reply:
Oh! Right. Good point. 👍